cli_tools/common/utils/logging/service/logger.go

```diff
@@ -178,6 +178,9 @@ func (l *Logger) getOutputInfo(w *daisy.Workflow, err error) *OutputInfo {
 	if err != nil {
 		o.FailureMessage = getFailureReason(err)
 		o.FailureMessageWithoutPrivacyInfo = getAnonymizedFailureReason(err)
+		if w != nil && w.Logger != nil {
```

Comment:
Due to serial logs being potentially large, it would good if we had a flag/config that would control if serial console logs are stored (it can default to true).

Reply:
We don't save logs if the use has specified [DisableCloudLogging](https://github.com/GoogleCloudPlatform/compute-image-tools/search?q=DisableCloudLogging&unscoped_q=DisableCloudLogging). Or were you thinking of something else?